modules/date_converter.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+import calendar
+
+
+def convert_month_to_number(month : str):
+    try:
+        if 0 < int(month) < 13:
```

Comment:
tá 13 je teraz magická konštanta, mohol by si to priradiť do nejakej premennej, ktorý by vyjadrila význam toho čísla?